plantuml2mysql.py

```diff
@@ -35,9 +33,14 @@ def main():
         print("Cannot open file: '" + sys.argv[1] + "'")
         sys.exit()
     # Add information for future self ;-)
-    print("# Database created on", time.strftime('%d/%m/%y %H:%M',time.localtime()), "from", sys.argv[1])
-    print("CREATE DATABASE %s CHARACTER SET = utf8 COLLATE = %s;" % (sys.argv[2], CHARSET))
-    print("USE %s;\n" % sys.argv[2])
+    print('')
+    print('')
+    print('####################################################################################################')
+    print("****************************\t TDH Info Model DB Script")
```

Comment:
Sorry, could you explain TDH acronym?

Reply:
TDH can be totally ignored. It is dome term sued in my project, forgot to remove.